frontend/src/views/pages/inventory/Inventory.vue

```diff
@@ -39,121 +28,65 @@
         />
       </CCol>
     </CRow>
+    <CModal
+      :show.sync="viewVehicle"
+      :centered="false"
+      title="Vehicle Information Page"
+      size="xl"
+    >
+      <vehicle v-if="!!$route.query.vehicleSelected" :vehicleId="$route.query.vehicleSelected"/>
+      <template #header>
+        <h6 class="modal-title">Vehicle Information Page</h6>
+        <CButtonClose @click="closeModal" />
+      </template>
+      <template #footer>
+        <span></span>
+      </template>
+    </CModal>
   </div>
 </template>
 
 <script>
 const axios = require("axios");
 import InventoryTable from "./InventoryTable.vue";
+import dealershipDD from "./DealershipDropdown.vue";
+import Vehicle from "../vehicle/Vehicle.vue"
 
 export default {
   name: "Inventory",
   data() {
     return {
-      tableItems: [],
-      tableFields: [],
-      adminDealerships: [],
       selectedDealership: null,
-      defaultAdminDealership: null,
       Message: null,
-      MessageType: null,
+      MessageType: null
     };
   },
+  computed: {
+    viewVehicle() {
+      return !!this.$route.query.vehicleSelected;
+    }
+  },
   methods: {
+    closeModal() {
+      let queries = JSON.parse(JSON.stringify(this.$route.query));
+      queries = {};
+      this.$router.replace({query: queries});
+    },
+    showQueryParams() {
+      console.log(this.$route.query);
+    },
     showMessage(message) {
-      this.Message = message;
+      this.Message = message.message;
       if (message.includes('error'))
-        this.MessageType = 'danger';
+        this.MessageType = message.messageType;
       else
```

Comment:
These if statements do not make sense. Please revise this code.

Reply:
Addressed in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/57/commits/0af7803e90afb19f20de7e446464ad1fcf5405f1)